common/src/test/java/org/apache/sedona/common/FunctionsTest.java

```diff
@@ -66,6 +66,81 @@ private Coordinate[] coordArray3d(double... coordValues) {
         return coords;
     }
 
+    @Test
+    public void asEWKT() throws Exception{
+        Geometry geometry = GEOMETRY_FACTORY.createPoint(new Coordinate(1.0, 2.0));
+        String actualResult = Functions.asEWKT(geometry);
```

Comment:
Shouldn't the test cases for EWKT have SRID too? @yyy1000 @jiayuasu

Reply:
Good idea.
I added SRID in them.